opencga-app/app/scripts/azure/arm/appgw/azuredeploy.json

```diff
@@ -0,0 +1,246 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "applicationGatewaySubnetId":{
+            "type": "string"
+        },
+        "skuName":{
+            "defaultValue": "Standard_Small",
+            "type": "string"
+        },
+        "skuTier":{
+            "defaultValue": "Standard",
+            "type": "string"
+        },
+        "skuCapacity":{
+            "defaultValue": 1,
+            "type": "int"
+        }      
+    },
+    "variables": {
+        "applicationGatewayName": "appgw",
+        "applicationGatewayPublicIpName": "appgwpublicip",
+        "applicationGatewayPublicIpId": "[resourceId('Microsoft.Network/publicIPAddresses',variables('applicationGatewayPublicIpName'))]",
+        "applicationGatewayId": "[resourceId('Microsoft.Network/applicationGateways', variables('applicationGatewayName'))]",
+        "skuV2": {
+                "name": "Standard_v2",
+                "tier": "Standard_v2",
+                "capacity": 1
+        }
+        
+
+    },
+    "resources": [{
+        "type": "Microsoft.Network/publicIPAddresses",
+        "name": "[variables('applicationGatewayPublicIpName')]",
+        "apiVersion": "2018-08-01",
+        "location": "[resourceGroup().location]",
+        "sku": {
+            "name": "Basic"
+        },
+        "properties": {
+            "publicIPAllocationMethod": "Dynamic"

```

Comment:
Probably should go `static` to minimize DNS churn

Reply:
v1 must have dynamic IP, Known limitation.

Reply:
well that's no fun